superbench/runner/runner.py

```diff
@@ -441,11 +441,11 @@ def _run_proc(self, benchmark_name, mode, vars):
             int: Process return code.
         """
         mode.update(vars)
-        if mode.name == 'mpi' and mode.pattern:
+        if mode.name == 'mpi' and 'pattern' in mode:
```

Comment:
mode.name always exists?

Reply:
Yes,  according to the schema of mode here: https://github.com/microsoft/superbenchmark/blob/main/docs/superbench-config.mdx#:~:text=parallel%3A%20yes-,name,-Mode%20name%20to